rel/overlay/etc/default.ini

```diff
@@ -753,3 +753,13 @@ port = {{prometheus_port}}
 ; to disable this setting could be if the views need an upgrade but located on
 ; read-only file system.
 ;commit_on_header_upgrade = true
+
+[custodian]
+; When set to `true`, force using `[cluster] n` values as the expected n value
+; of of shard copies. In cases where the application prevents creating
```

Comment:
Maybe replace "the application" with "custodian" for clarity?

Reply:
In this case it would be a the user's application which uses CouchDB underneath. Say, if they'd never be a chance to have any databases with `n=2` or `n=1`. But, there is a chance the shard map could be manipulated by hand (rebalancing shards, restoring backups, etc) and so we'd want custodian to warn about some missing copies. With the current scheme of taking n = `number of shard ranges in the shard map` it's impossible to check for that condition (but, instead we allow creating dbs with n < cluster n and don't flag that as an error in custodian).